src/tests/java/com/util/CodeUtilTest.java

```diff
@@ -0,0 +1,71 @@
+package com.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CodeUtilTest {
+
+    @Test
+    public void testEscape() {
+        Assert.assertEquals("%80%u9e00Ё%00A",
+                CodeUtil.escape("\u0080\u9e00\u0401\u0000A"));
+    }
+
+    @Test
+    public void testUnescape() {
+        Assert.assertEquals("fo\u0011ar", CodeUtil.unescape("fo%11ar"));
+    }
+
+    @Test
+    public void testDbEncode() {
+        Assert.assertNull(CodeUtil.dbEncode(null));
+        Assert.assertEquals("foo", CodeUtil.dbEncode("foo"));
+    }
+
+    @Test
+    public void testDbDecode() {
+        Assert.assertNull(CodeUtil.dbDecode(null));
+        Assert.assertEquals("foo", CodeUtil.dbDecode("foo"));
```

Comment:
should have some tests for `dbEncode` and `dbDecode` which show characters being encoded/decoded

Reply:
Done